### ISSUE TITLE
Does not compile

### ISSUE THREAD

**Issue by Jerther** [teklanology/Tekla-Properties#1]
I get 12 errors like this one:
`Cannot implicitly convert type 'Tekla.Structures.Drawing.DrawingHatchColors' to 'Tekla.Structures.Drawing.DrawingColors'. An explicit conversion exists (are you missing a cast?)	TeklaProperties	C:\Users\Principal\Source\Repos\Tekla-Properties\17.0\DrawingGraphic.cs	583`
I get the same errors with the 16.1 version.

**Comment by dalenicholls** [comment]
have you modified the code? i can't find no reference to DrawingHatchColors, it should be DrawingColors?

**Comment by Jerther** [comment]
Code is as is.
Here, the type of `rectangle.Attributes.Hatch.BackgroundColor` at https://github.com/teklanology/Tekla-Properties/blob/master/17.0/DrawingGraphic.cs#L583 is `DrawingHatchColors`.

![image](https://user-images.githubusercontent.com/5856745/31548543-291ec1be-aff9-11e7-8576-1313f0cff4d4.png)

**Comment by dalenicholls** [comment]
its defined as DrawingColors on https://github.com/teklanology/Tekla-Properties/blob/master/17.0/DrawingGraphic.cs#L482.
what version of tekla's dll are you using?

**Comment by Jerther** [comment]
2016.1.8961.0
But it also happens if I switch to 2017.1.10074

**Comment by dalenicholls** [comment]
16.1 and 17.0 refer to tekla version not year. your references are more recent than mine, hence why the hatch.backgroundcolor is trying to force to a drawingcolor. try changing line 482 to read the following

```
public DrawingHatchColors HatchBackgroundColour
```

**Comment by Jerther** [comment]
Sorry you're right about the version, they should be 16 and 17. I guess I took the version in the property dialog and appended the two last digits from the file version. My bad :)

![image](https://user-images.githubusercontent.com/5856745/31552787-69908eaa-b006-11e7-9a56-bca6049549d4.png)

So what version are you using? The .csproj files for 16.1 and 17.0 indicate 16.1 and 17.0 respectively but also have all SpecificVersion tags set to False

https://github.com/teklanology/Tekla-Properties/blob/master/16.1/TeklaProperties.csproj#L37

I did change all `DrawingColors` to `DrawingHatchColors` in `DrawingGraphic.cs` and, well... It builds now :)
